api/graphql/types/allocated_time_slot/permissions.py

```diff
@@ -36,17 +36,12 @@ def has_create_permission(cls, user: AnyUser, input_data: dict[str, Any]) -> boo
             msg = "Reservation Unit Option is required for creating an Allocated Time Slot."
             raise GQLCodeError(msg, code=error_codes.REQUIRED_FIELD_MISSING)
 
-        option: ReservationUnitOption | None = (
-            ReservationUnitOption.objects.filter(pk=option_pk)
-            .select_related("application_section__application__application_round__service_sector")
-            .first()
-        )
+        option: ReservationUnitOption | None = ReservationUnitOption.objects.filter(pk=option_pk).first()
         if not option:
             msg = f"Reservation Unit Option with pk {option_pk} does not exist."
             raise GQLCodeError(msg, code=error_codes.ENTITY_NOT_FOUND)
 
-        sector = option.application_section.application.application_round.service_sector
-        return can_manage_service_sectors_applications(user, sector)
+        return can_manage_service_sectors_applications(user)
```

Comment:
Let's replace the `can_manage_service_sectors_applications` check with the user having the `CAN_HANDLE_APPLICATIONS` permission on the unit (of the reservation unit) of the option which is being allocated.

Reply:
Done.